### PR TITLE
Add support for ansible-lint to v6

### PR DESCRIPTION
**Pull request by ssbarnea**
Issue: AAH-1429

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -17,7 +17,7 @@ packages = find:
 install_requires =
     ansible-core
     ansible-builder>=1.0.1,<2.0
-    ansible-lint>=5.0.8,<6.0
+    ansible-lint>=6.0.0,<7.0
```
Comment (Spredzy) [inline setup.cfg:20]: Why the jump to `>=6.0` and not leave `>=5.0` ? Any technical reason ?

Comment (ssbarnea) [reply setup.cfg]: They diverged considerably and will produce different results. We can keep it like this but you should be aware that ansible-lint upstream will not provide support for 5.x.

Ideally, you should look into removing support for py37.

**Comment by awcrosby** [comment]
@ssbarnea the ci `build-and-test` fail issue that was at HEAD has been resolved. If you rebase then ci should run as expected.

**Comment by awcrosby** [comment]
@ssbarnea I clicked rebase on this PR and the tests are passing, For commit check failure take a look at https://github.com/ansible/galaxy-importer/blob/master/PROCESS.md#issues-and-pull-requests

File: setup.cfg [hunk 2]
```diff
@@ -17,7 +17,8 @@ packages = find:
 install_requires =
     ansible-core
     ansible-builder>=1.0.1,<2.0
-    ansible-lint>=5.0.8,<6.0
+    ansible-lint>=5.4.0,<6.0; python_version<'3.8'
+    ansible-lint>=6.0.0,<7.0; python_version>='3.8'
```
Comment (awcrosby) [inline setup.cfg:21]: @ssbarnea I think we should keep it so galaxy-importer supports one range of ansible-lint

**Review by awcrosby — CHANGES_REQUESTED** [review]

**Comment by awcrosby** [comment]
closing via #164

**Pull request closed without merging**